### PR TITLE
#54: Added OpenSuse42.1 and OpenSuse15.3 CI env tests

### PR DESCRIPTION
**Pull request by tomuben**

### CODE REVIEW

File: ci_tests/OpenSuse15.3/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM opensuse/leap:15.3
+
+RUN zypper --non-interactive update
```
Comment (tkilias) [inline ci_tests/OpenSuse15.3/Dockerfile:3]: Do we need the update?

File: ci_tests/OpenSuse42.1/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,7 @@
+FROM opensuse/archive:42.1
+
+RUN zypper --non-interactive update
```
Comment (tkilias) [inline ci_tests/OpenSuse42.1/Dockerfile:3]: Do we need the update?

**Review by tkilias — COMMENTED** [review]
Except the update question, everything is fine

File: ci_tests/OpenSuse42.1/Dockerfile (hunk 2 above)
Comment (tomuben) [reply ci_tests/OpenSuse42.1/Dockerfile:3]: I think it runs the update automatically if we don't call it explicitly.

**Pull request merged**